src/goto-checker/solver_factory.cpp

```diff
@@ -357,7 +357,7 @@ solver_factoryt::get_smt2(smt2_dect::solvert solver)
       ns,
       "cbmc",
       std::string("Generated by CBMC ") + CBMC_VERSION,
-      "QF_AUFBV",
+      "ALL",
```

Comment:
:pick: The title of this commit states that the change is only being make for `cvc4`, but the code changes appear to be changing it for all solvers.